.neutrinorc.js

```diff
@@ -1,9 +1,11 @@
+const eslint = require('@neutrinojs/eslint');
+const react = require('@neutrinojs/react');
+const copy = require('@neutrinojs/copy');
+const jest = require('@neutrinojs/jest');
```

Comment:
The new style syntax still supports inlining the presets if desired (it then saves installing/loading them in production) - it's just that the migrate tool isn't advanced enough to recognise that style.

eg instead of:
```js
    process.env.NODE_ENV === 'development' &&
      eslint(
```

you can use:

```js
    process.env.NODE_ENV === 'development' &&
      require('@neutrinojs/eslint')(
```

(you might also need to add an `// eslint-disable-next-line ...` about global requires)